lms/djangoapps/bulk_email/models.py

```diff
@@ -215,6 +216,10 @@ def render_htmltext(self, htmltext, context):
         Convert HTML text body (`htmltext`) into HTML email message using the
         stored HTML template and the provided `context` dict.
         """
+        # HTML-escape string values in the context (used for keyword substitution).
+        for key, value in context.iteritems():
```

Comment:
Done in https://github.com/edx/edx-platform/pull/12335/commits/d55d128907ad8694518834137228394433eabbe4. Note that there was a small difference in the encoding.